diff --git a/meson.build b/meson.build
index d0af9ee..cfbaadc 100644
--- a/meson.build
+++ b/meson.build
@@ -97,10 +97,7 @@ libsecret = dependency('libsecret-1', version: '>= 0.11')
 libsoup = dependency('libsoup-3.0')
 libstemmer_dep = cc.find_library('stemmer')
 libunwind_dep = dependency(
-  'libunwind', version: '>= 1.1', required: get_option('libunwind')
-)
-libunwind_generic_dep = dependency(
-  'libunwind-generic', version: '>= 1.1', required: get_option('libunwind')
+  'libunwind-llvm', version: '>= 1.1', required: get_option('libunwind')
 )
 libxml = dependency('libxml-2.0', version: '>= 2.7.8')
 libytnef = dependency('libytnef', version: '>= 1.9.3', required: get_option('tnef'))
@@ -126,7 +123,6 @@ if libunwind_dep.found()
      dependencies: [
        valac.find_library('libunwind', dirs: [vapi_dir, unwind_lib]),
        cc.find_library('libunwind', dirs: unwind_lib),
-       cc.find_library('libunwind-generic', dirs: unwind_lib)
      ],
    )
 endif
