diff -up geary-3.36.2/meson.build.1~ geary-3.36.2/meson.build
--- geary-3.36.2/meson.build.1~	2020-05-03 10:26:43.017199800 +0200
+++ geary-3.36.2/meson.build	2020-06-02 18:03:07.302248767 +0200
@@ -87,9 +87,6 @@ libsoup = dependency('libsoup-2.4', vers
 libunwind_dep = dependency(
   'libunwind', version: '>= 1.1', required: not get_option('libunwind_optional')
 )
-libunwind_generic_dep = dependency(
-  'libunwind-generic', version: '>= 1.1', required: not get_option('libunwind_optional')
-)
 libxml = dependency('libxml-2.0', version: '>= 2.7.8')
 libytnef = dependency('libytnef', version: '>= 1.9.3', required: get_option('tnef-support'))
 posix = valac.find_library('posix')
@@ -110,7 +107,6 @@ if libunwind_dep.found()
      dependencies: [
        valac.find_library('libunwind', dirs: [vapi_dir, unwind_lib]),
        cc.find_library('libunwind', dirs: unwind_lib),
-       cc.find_library('libunwind-generic', dirs: unwind_lib)
      ],
    )
 endif
